functions/tests/actions/wayback-machine.spec.js

```diff
@@ -0,0 +1,64 @@
+const axios = require('axios');
+const {expect} = require('chai');
+const rewire = require('rewire');
+
+const action = rewire('../../src/actions/wayback-machine');
+
+const mockApp = require('../_utils/mocking/platforms/app');
+const mockDialog = require('../_utils/mocking/dialog');
+
+describe('actions', () => {
+  describe('wayback machine', () => {
+    let app;
+    let dialog;
+    let archiveRequest = axios.get('http://web.archive.org/__wb/search/metadata?q=cnn.com');
```

Comment:
We can't request data for a test from an ourside. If you would need any data you should create fixture data.

Examples: https://github.com/internetarchive/internet-archive-google-action/tree/master/functions/tests/fixtures